suites/apis/rasCronjobValidate.js

```diff
@@ -0,0 +1,19 @@
+Feature('RAS cronjob validation');
+
+const { expect } = require('chai');
+const { interactive, ifInteractive } = require('../../utils/interactive.js');
+
+const TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME;
+
+Scenario('Validate fence-visa-update job @manual', ifInteractive(
+  async () => {
+    const result = await interactive(`
+            1. Get a new passport from /userinfo endpoint
+            2. Log into ${TARGET_ENVIRONMENT} with RAS Login (RAs user and password)
+            3. run `gen3 job run fence-visa-update`
```

Comment:
should check visa table before job is run, note number of visas for your user - perhaps note the JTI and expiration

Reply:
👍  let me add that steps and also some description on what needs to be done here while checking the db